aspnet/mobile/native-mobile-backend.rst

```diff
@@ -1,11 +1,170 @@
+
+.. image:: native-mobile-backend/_static/todo-android.png
+
+.. note:: Securing web APIs is not covered in this article, but will be covered in follow-up documentation.
+
+Features
+^^^^^^^^
+
+The TodoRest app supports listing, adding, deleting, and updating To-Do items. Each item has an ID, a Name, Notes, and a property indicating whether it's been Done yet.
+
+The main view of the items, as shown above, lists each item's name and indicates if it is done with a checkmark.
+
+Tapping the `+` icon opens an add item dialog:
+
+.. image:: native-mobile-backend/_static/todo-android-new-item.png
+
+Tapping an item on the main list screen opens up an edit dialog where the item's Name, Notes, and Done settings can be modified, or the item can be deleted:
+
+.. image:: native-mobile-backend/_static/todo-android-edit-item.png
+
+This sample works with a read-only set of backend services by default. To test it out yourself against the ASP.NET Core app created in the next section running on your computer, you'll need to update the app's ``RestUrl`` constant. Navigate to the ``TodoREST`` project and open the `Constants.cs` file. Replace the ``RestUrl`` with a URL that includes your machine's IP address (not localhost or 127.0.0.1, since this address is used from the device emulator, not from your machine). Include the port number as well (5000). In order to test that your services work with a device, ensure you don't have an active firewall blocking access to this port.
+
+.. code-block:: csharp
+
+    // URL of REST service (Xamarin ReadOnly Service)
+    //public static string RestUrl = "http://developer.xamarin.com:8081/api/todoitems{0}";
+    
+    // use your machine's IP address
+    public static string RestUrl = "http://192.168.1.207:5000/api/todoitems/{0}";
+
+
+Creating the ASP.NET Core MVC Project
+-------------------------------------
+
+Create a new ASP.NET Core MVC project in Visual Studio. Choose the Web API template and No Authentication.
+
+.. image:: native-mobile-backend/_static/web-api-template.png
+
+The application should respond to all requests made to port 5000. Update `Program.cs` to include ``.UseUrls("http://*:5000")`` to achieve this:
+
+.. literalinclude:: native-mobile-backend/sample/ToDoApi/src/ToDoApi/Program.cs
+  :language: c#
+  :lines: 10-16
+  :dedent: 12
+  :emphasize-lines: 3
+
+.. note:: Make sure you run the application using Kestrel, not IIS Express. This can be done from Visual Studio, or by running ``dotnet run`` from the command line.
```

Comment:
I think this should be reworded to say that when running the sample locally you should run the app directly instead of running it behind IIS Express. Technically your app runs on Kestrel in both cases - what matters is that you don't want to use IIS Express.